rpcserver.go

```diff
@@ -6898,6 +6899,7 @@ func (r *rpcServer) createBackupSnapshot(backups []chanbackup.Single) (
 	var b bytes.Buffer
 	unpackedMultiBackup := chanbackup.Multi{
 		StaticBackups: backups,
+		Encrypter:     lnencrypt.Encrypter{},
```

Comment:
It seems like you missed one in `PackStaticChanBackups` when creating the `*Single`, which is why I think not letting `Encrypter` be a member is a good idea